client/src/document/ingredients/browser-compatibility-table/index-mobile.scss

```
@@ -25,7 +25,7 @@
     }
   }
 
-  .table-container {
+  .bc-table-outer {
```

Mobile view needs following changes
```suggestion
  .table-container{
      overflow-x: auto;
    }
```

https://github.com/mdn/yari/assets/91728992/d3c3b736-8f01-4a0f-a1ed-0fb09eb9021a